### DISCUSSION TITLE
Accessing the object tracker

### DISCUSSION THREAD

**Discussion by noahweber1-debug** [talmolab/sleap#929] [General]
After one tracks a certain object. Is there a possibility to access all of these individual tracked objects and use the centroid (or whatever piece of the pose estimation) do to further calculations on the resulting videos?

So the question is not how to access the resulting videos, but rather tracked objects in the resulting videos. If there is a tutorial pointing to this it would be even better.

**Comment by talmo** [comment]
Hi @noahweber1-debug,

Sure thing -- of particular interest here would be our notebooks on SLEAP's [Data structures](https://sleap.ai/notebooks/Data_structures.html) and [Analysis examples](https://sleap.ai/notebooks/Analysis_examples.html).

Let us know if those don't cover what you had in mind.

Cheers,

Talmo
